indico/modules/events/roles/controllers.py

```diff
@@ -83,6 +84,7 @@ class RHManageEventRole(RHManageEventBase):
     def _process_args(self):
         RHManageEventBase._process_args(self)
         self.role = EventRole.get_one(request.view_args['role_id'])
+        self.logger = logger
```

Comment:
You don't need this here, since you already have access to `self.logger` in the classes that need it.

Reply:
Ohh, I forgot to remove it.